docs/architecture/observers.mdx

```diff
@@ -0,0 +1,32 @@
+---
+sidebar_position: 9
+---
+
+import ObserverList from "@site/src/components/ObserverList/ObserverList";
+import ObserverParams from "@site/src/components/ObserverParams/ObserverParams";
+
+# Observers
```

Comment:
I think since they are coupled, could we call it Observer-Signer throughout this page?